### PR TITLE
feat(frontend): GitHub Guide 257

### LINKED ISSUE #257: Add GitHub Tutorial
https://github.com/firstcontributions/first-contributions

### PR DESCRIPTION
**Pull request by steph222**
### GH Issue

Resolves #257 

### Steps to test
1. Go to GitHub Guide
2. Check content is right.

#### CheckList
- [ ] Follow proper Markdown format
- [ ] The content is adequate
- [ ] The content is equal to https://guias.eoscostarica.io/docs/tutoriales/guia-github

### CODE REVIEW

File: docs/tutorials/github-guide.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+[GitHub for mobile](https://play.google.com/store/apps/details?id=com.github.android&hl=es_419&gl=US) is available for Android and iOS, it gives you a way to get high-impact work on GitHub quickly, from anywhere. It also becomes a secure and stable way to access GitHub data through a trusted source client application.
+
+<img alt="GitHub Móvil Image" src={ useBaseUrl( 'img/LogosGitHub/GitHub-Movil.png' )} />
+
+### [Git Bash](https://desarrolloweb.com/articulos/entiende-instala-configura-git.html#:~:text=Git%20Bash%20es%20la%20línea,para%20usar%20Git%20en%20Windows.)
+
+[Git Bash](https://gitforwindows.org) is a Git application for Windows, where it allows working through command lines managing projects found on GitHub.
+
+<img alt="Git Bash Image" src={ useBaseUrl( 'img/LogosGitHub/GitHub-Bash.png' )} />
+
```
Comment (xavier506) [inline docs/tutorials/github-guide.md:41]: There are also tools like Hub and git-extras that facilitate interacting with Github. You can leverage these tools to contribute to this repository.

---

Hay también herramientas como Hub y git-extras que facilitan la interacción con GitHub.

Please see : https://guias.eoscostarica.io/docs/pautas-para-codigo-abierto#herramientas-de-git-avanzadas

**Comment by xavier506** [comment]
@steph222  Looking good ! 

I found this other resource to be interesting for those who are getting started with git / github 

perhaps it is worth including : 
https://github.com/firstcontributions/first-contributions

I know we already merged the spanish version so we may have to create a new PR to bring any updates over to the spanish version.

File: docs/tutorials/github-guide.md (hunk 1 above)
Comment (steph222) [reply docs/tutorials/github-guide.md:41]: Of course!!

**Review by xavier506 — APPROVED** [review]
LGTM

**Review by Edgar097 — APPROVED** [review]

**Review by leisterfrancisco — APPROVED** [review]
Good work 👍

**Pull request merged**